package.json

```diff
@@ -25,7 +25,7 @@
     "url": "https://github.com/roadmanfong/react-cropper.git"
   },
   "dependencies": {
-    "cropperjs": "v1.0.0-rc.3",
+    "cropperjs2": "^2.0.0",
```

Comment:
Hello @yujiangshui. I have been trying to lookup [cropperjs2](https://www.npmjs.com/package/cropperjs2), but cannot seem to find the change log from [cropperjs](https://www.npmjs.com/package/cropperjs). How are they different besides the only difference that I can find that cropperjs2 omits [initialAspectRatio](https://www.npmjs.com/package/cropperjs#initialaspectratio) from its options.

I also see you published [react-cropper2](https://www.npmjs.com/package/react-cropper2), but like cropperjs2, the documentation and repo all point back to the previous version without a descriptive change log.
How are these packages different.

Reply:
I think @yujiangshui should maintain their own `@yujiangshui/react-cropper`, so I've add -1 to this PR.